build/proxysql.cnf

```diff
@@ -0,0 +1,30 @@
+admin_variables=
+{
+	admin_credentials="admin:admin;radmin:radmin"
+}
+
+# defines all the MySQL servers
+mysql_servers =
+(
+	{ address="host.docker.internal" , port=3306 , hostgroup=1, comment="Source" },
+	{ address="host.docker.internal" , port=3307 , hostgroup=2, weight=1000, comment="Readyset" },
+	{ address="host.docker.internal" , port=3306 , hostgroup=2, weight=1, comment="Source" }
+)
+
+mysql_variables=
+{
+	monitor_username="root"
+	monitor_password="noria"
+}
+
+mysql_users:
```

Comment:
nit: afaict `:` is equivalent to `=` so we could make this more consistent. Similarly, may as well be consistent about spaces around `=`.